newproject01/newproject01/q1.c

```diff
@@ -0,0 +1,67 @@
+#define _CRT_SECURE_NO_WARNINGS
+#include <stdio.h>
+#define N 8
+
+isNumberInOrder(int[], int, int); /*Declaration of a function*/
```

Comment:
c function names should be in snake_case. is_number_in_order(int[], int, int)

Reply:
part 1 done
part 2 I think we didn't able to do it yet